x-pack/legacy/plugins/apm/public/context/LicenseContext/index.tsx

```diff
@@ -5,16 +5,13 @@
  */
 import React from 'react';
 import { FETCH_STATUS, useFetcher } from '../../hooks/useFetcher';
-import { loadLicense } from '../../services/rest/xpack';
+import { loadLicense, LicenseApiResponse } from '../../services/rest/xpack';
 import { InvalidLicenseNotification } from './InvalidLicenseNotification';
 
 const initialLicense = {
-  features: {
-    watcher: { is_available: false },
-    ml: { is_available: false }
-  },
+  features: {},
   license: { is_active: false }
-};
+} as LicenseApiResponse;
```

Comment:
should we not use initialLicense:LicenseAPIResponse?

Reply:
Yes, but since the other items in license were required I'd have to add default value for those as well:
```ts
license: {
    expiry_date_in_millis: number;
    is_active: boolean;
    type: string;
}
```

Since we don't use those two properties I'll just remove them.

Reply:
Fixed